pom.xml

```diff
@@ -604,6 +610,16 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
I don't know if we want to activate it by default or not ?
We can move the configuration to `pluginManagement` if not and let projects choose to use it explicitly 😇.

Reply:
maybe you right ... it will remove vote process at all

another idea to introduce property, eg: `autoRelease` or something else - default to `false`
when project want such simple change property value.

one benefit what I see is that repository will be closed after release, now we have two actions: close and release.

for reference: https://www.mojohaus.org/development/performing-a-release.html#Staging_the_Release